src/common/components/dashboard/dashboard.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+const styles: any = require('./dashboardStyles.scss');
+import {DashboardItemComponent, IDashboardItem} from './components/dashboardItem';
+
+interface IProps {
+  title: string;
+  items: IDashboardItem[];
+}
+
+export class DashboardComponent extends React.Component<IProps, {}> {
+  public render() {
+    return(
+      <div className={styles.dashboard}>
+        <h3 className={styles.dashboardTitle}>{this.props.title}</h3>
```

Comment:
I think dashboardTitle does not apply to this component, just the page should display the title